src/std/rpc/ws_client/mod.rs

```diff
@@ -141,149 +143,113 @@ where
 	}
 }
 
-pub fn on_get_request_msg(msg: Message, out: Sender, result: ThreadOut<String>) -> WsResult<()> {
```

Comment:
The diff from here on is a little messed up.
The `on_get_request_msg` has not changed (only moved to be implemented by `GetRequestHandler`, same goes for the `on_subscription_msg`). The main change: `SubmitAndWatchHandler` now substitutes all the `on_extrinsic_msg_until_*` functions.